ui/src/admin/components/chronograf/UsersTable.js

```diff
@@ -0,0 +1,155 @@
+      return false
+    }
+    const {isSameUser} = this.props
+    return !_.differenceWith(usersA, usersB, isSameUser).length
+  }
+
+  render() {
+    const {
+      organization,
+      organizations,
+      filteredUsers,
+      onToggleAllUsersSelected,
+      onToggleUserSelected,
+      selectedUsers,
+      isSameUser,
+    } = this.props
+    const {colOrg, colRole, colSuperAdmin, colProvider, colScheme} = USERS_TABLE
+
+    const areAllSelected = this.areSameUsers(filteredUsers, selectedUsers)
+
+    return (
+      <table className="table table-highlight v-center chronograf-admin-table">
+        <thead>
+          <tr>
+            <th className="chronograf-admin-table--check-col">
+              <div
+                className={
+                  areAllSelected ? 'user-checkbox selected' : 'user-checkbox'
+                }
+                onClick={onToggleAllUsersSelected(areAllSelected)}
+              />
+            </th>
+            <th>Username</th>
+            <th style={{width: colOrg}}>Organization</th>
+            <th style={{width: colRole}}>Role</th>
+            <Authorized requiredRole={SUPERADMIN_ROLE}>
+              <th style={{width: colSuperAdmin}}>SuperAdmin</th>
+            </Authorized>
+            <th style={{width: colProvider}}>Provider</th>
+            <th className="text-right" style={{width: colScheme}}>
+              Scheme
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {filteredUsers.length
+            ? filteredUsers.map(
+                (user, i) =>
+                  organization.name === DEFAULT_ORG_NAME
+                    ? <UsersTableRow
+                        user={user}
+                        key={i}
+                        organizations={organizations}
+                        onToggleUserSelected={onToggleUserSelected}
+                        selectedUsers={selectedUsers}
+                        isSameUser={isSameUser}
+                        onSelectAddUserToOrg={this.handleSelectAddUserToOrg(
+                          user
+                        )}
+                        onChangeUserRole={this.handleChangeUserRole}
+                        onChooseFilter={this.handleChooseFilter}
+                        onChangeSuperAdmin={this.handleChangeSuperAdmin}
+                      />
+                    : <OrgTableRow
+                        user={user}
+                        key={i}
+                        onToggleUserSelected={onToggleUserSelected}
+                        selectedUsers={selectedUsers}
+                        isSameUser={isSameUser}
+                        organization={organization}
+                        onSelectAddUserToOrg={this.handleSelectAddUserToOrg(
+                          user
+                        )}
+                        onChangeUserRole={this.handleChangeUserRole}
+                      />
+              )
+            : <tr className="table-empty-state">
+                <Authorized
```

Comment:
This looks like it's only controlling the width of the resulting `th`

Reply:
correct -- superadmin th is also hidden, and this goes along with it